DotNetOnion.Tests/DirectoryParsers.cs

```diff
@@ -4,37 +4,26 @@
 using System.Net;
 using System.Text;
 using System.Threading.Tasks;
-using Microsoft.FSharp.Core;
-
 using NUnit.Framework;
 
 using NOnion.Directory;
-using NOnion.Http;
-using NOnion.Network;
-
+using Newtonsoft.Json;
+using System.IO;
 
 namespace DotNetOnion.Tests
 {
     public class DirectoryParsers
     {
-        private readonly IPEndPoint torServer = IPEndPoint.Parse("199.184.246.250:443");
-
         [Test]
-        public async Task CanParseNetworkStatusDocumentAsync ()
+        public void CanParseNetworkStatusDocumentAndConvertToJson ()
         {
-            using TorGuard guard = await TorGuard.NewClientAsync(torServer);
-            using TorCircuit circuit = new(guard);
-            using TorStream stream = new(circuit);
-
-            await circuit.CreateFastAsync();
-            await stream.ConnectToDirectoryAsync();
-
-            TorHttpClient httpClient = new TorHttpClient(stream, torServer.Address.ToString());
-            var response = await httpClient.GetAsStringAsync("/tor/status-vote/current/consensus");
+            var networkStatusStr = System.IO.File.ReadAllText($"Directory-Samples{Path.DirectorySeparatorChar}NetworkStatus.txt");
+            var expectedNetworkStatusJson = System.IO.File.ReadAllText($"Directory-Samples{Path.DirectorySeparatorChar}NetworkStatus.json");
 
-            NetworkStatusDocument networkStatus = NetworkStatusDocument.Parse(response);
+            NetworkStatusDocument networkStatus = NetworkStatusDocument.Parse(networkStatusStr);
+            var networkStatusJson = JsonConvert.SerializeObject(networkStatus, Formatting.Indented);
 
-            Assert.IsTrue(FSharpOption<int>.get_IsSome(networkStatus.Version));
```

Comment:
nit: let's use `Assert.That...`, this way it's less likely to swap the expected thing with the actual thing